### ISSUE TITLE
False positive reportUnusedVariable

### ISSUE THREAD

**Issue by lovetox** [microsoft/pyright#5841]
**Describe the bug**
fields is reported with

`error: Variable "fields" is not accessed (reportUnusedVariable)`

altough the variable is used in the next line

**Code**

```python
from typing import Any
from collections import namedtuple

def namedtuple_factory() -> Any:
    fields = ["a", "b", "c"]
    Row = namedtuple('Row', fields)
    return Row
```

**VS Code extension or command-line**
pyright 1.1.324 on command line

**Comment by erictraut** [comment]
This is included in pyright 1.1.325, which I just published. It will also be included in an upcoming pylance release.
